_projects/kazv_meta.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Tackle a meta issue in kazv
+desc: Add more features to kazv to make it more usable
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - C++ or JavaScript
+ - Knowledge of Qt, Qml and kde frameworks is a plus
+ - Use, or be willing to use GNU/Linux (e.g. Ubuntu) or musl/busybox Linux (e.g. Alpine)
+difficulty: medium
+issues:
+# Related issues (if any)  to this project.
+ - https://lily.kazv.moe/kazv/kazv/-/issues/17
+ - https://lily.kazv.moe/kazv/kazv/-/issues/18
+ - https://lily.kazv.moe/kazv/kazv/-/issues/20
+ - https://lily.kazv.moe/kazv/kazv/-/issues/19
+ - https://lily.kazv.moe/kazv/kazv/-/issues/5
+mentors:
+# First person in contact; mentors may change before project starts.
+ - tusooa
+initiatives:
+ - GSoC
+tags:
```

Comment:
Can you add a tag with either 175h or 350h depending on how long you think the project would be?

Reply:
ummm, that can be hard, as stated in the document it is pretty flexible, so I cannot say how long it would take. but speaking of my own preferences, I want it to be shorter as I expect I would be super busy after summer. I'll add the short tag.